seatunnel-connectors-v2/connector-file/connector-file-base/src/test/java/org/apache/seatunnel/connectors/seatunnel/file/writer/ParquetReadStrategyTest.java

```diff
@@ -154,6 +167,32 @@ public void testParquetReadProjection2() throws Exception {
         parquetReadStrategy.read(path, "", testCollector);
     }
 
+    @Test
+    public void testParquetReadArray() throws Exception {
+        String os = System.getProperty("os.name").toLowerCase();
+        if (!os.contains("win")) {
```

Comment:
You can use `@DisabledOnOs(OS.WINDOWS)` to instead of this. Please refer https://github.com/apache/seatunnel/blob/6f49ec6ead82f842c6d93563ffce64853deb1bed/seatunnel-engine/seatunnel-engine-server/src/test/java/org/apache/seatunnel/engine/server/master/JobHistoryServiceTest.java#L39

Reply:
ok